devtools/common/src/main/java/io/quarkus/maven/utilities/MojoUtils.java

```diff
@@ -329,4 +335,21 @@ public Attribute(String name, String value) {
         }
     }
 
+    public static String[] readGavFromSettingsGradle(ByteArrayInputStream buildFileInputStream, String[] gavIn) {
+        String[] gavOut = Arrays.copyOf(gavIn, gavIn.length);
+        try (Scanner scanner = new Scanner(buildFileInputStream)) {
+            while (scanner.hasNextLine()) {
+                String currentLine = scanner.nextLine();
+                if (currentLine.startsWith("group")) {
```

Comment:
It looks like these property names here and in the template are not consistent, i.e. the `project.` prefix used in the template.